cosmo_tester/test_suites/cluster/compact_cluster_test.py

```diff
@@ -66,6 +66,16 @@ def test_three_nodes_cluster_teardown(three_nodes_cluster, ssh_key,
     _assert_cluster_status(node1.client)
 
 
+@pytest.mark.three_vms_ipv6
+def test_three_nodes_cluster_ipv6_status(three_nodes_ipv6_cluster, logger):
+    node1, node2, node3 = three_nodes_ipv6_cluster
```

Comment:
We should probably be deploying an agent as well (which I realise will make this not really just be a _status test, so maybe drop that suffix), using the ipv6 address of one of the managers.